icu.yaml

```diff
@@ -66,6 +66,5 @@ subpackages:
 
 update:
   enabled: true
-  manual: true # melange bump doesn't work with var-transforms yet, manual updates required
```

Comment:
Can't do this one yet without backwards mapping either.

Reply:
Alright, noted.